python/cuml/neighbors/ann.pyx

```diff
@@ -75,10 +75,11 @@ cdef build_ivfpq_algo_params(params, automated, additional_info):
                     params['M'] = n_subq
                     break
 
-        for i in reversed(range(1, 4)):
-            min_train_points = (2 ** i) * 39
+        params['n_bits'] = 4
```

Comment:
Why possible values for `n_bits` went from  {1, 2, 3} to {4, 5, 6, 8} ? Can you add a comment so that we better understand this block of code ?

Reply:
`n_bits` could go from 1 to 8 before FAISS update. With FAISS 1.7, n_bits must be in set {4, 5, 6, 8}. See [FAISS code](https://github.com/facebookresearch/faiss/blob/6977d72b1d6fde2085505b99199c6f95a3b2f10a/faiss/gpu/GpuIndexIVFPQ.cu#L404).